tests/checkpointing/test_model_checkpoint.py

```diff
@@ -261,6 +261,40 @@ def test_model_checkpoint_format_checkpoint_name(tmpdir):
     assert ckpt_name == filepath / 'test-epoch=3-step=2.ckpt'
 
 
+def test_model_checkpoint_file_extension(tmpdir):
+    
+    # tests that format_checkpoint_name uses the user-defined FILE_EXTENSION
+    ckpt_name = ModelCheckpoint(monitor='early_stop_on', dirpath='.').format_checkpoint_name(0, 1, {}) 
+    ModelCheckpoint.FILE_EXTENSION = '.tpkc'
+    tpkc_name = ModelCheckpoint(monitor='early_stop_on', dirpath='.').format_checkpoint_name(0, 1, {})
+    assert ckpt_name == str(Path('.').resolve() / 'epoch=0-step=1.ckpt')
+    assert tpkc_name == str(Path('.').resolve() / 'epoch=0-step=1.tpkc')
+    
+    #tests that _save_last_checkpoint uses the user-defined FILE_EXTENSION
+    seed_everything()
+    model = LogInTwoMethods()
+    epochs = 1
+    ModelCheckpoint.CHECKPOINT_NAME_LAST = 'last-{epoch}'
```

Comment:
also, no need for this, since it's already tested in the test below. just use the defaults here.